tests/neuro_san/zzz_hocons/test_smoke_test_hocons.py

```diff
@@ -42,7 +42,7 @@ class TestSmokeTestHocons(TestCase):
         "music_nerd_pro/combination_responses_with_history_direct.hocon",
 
         # List more hocon files as they become available here.
-    ]))
+    ]), skip_on_empty=True)
```

Comment:
Add skip_on_empty to the hocon test harnesses. This is a nice little trick I found that allows smooth running if all the tests happen to be commented out. It's good templating for future repos.